.goreleaser.yml

```diff
@@ -220,4 +262,53 @@ docker_manifests:
     image_templates:
       - "confluentinc/confluent-cli:latest-amd64"
       - "confluentinc/confluent-cli:latest-arm64"
-    skip_push: "{{ .Env.DRY_RUN }}"
\ No newline at end of file
+    skip_push: "{{ .Env.DRY_RUN }}"
+
+release:
+  ids:
+    - archive
+    - archive-alpine
```

Comment:
except alpine, all other linux builds are in archive now 💛

Reply:
Yep! I tried to get alpine in there too, but it looks like you can't have a duplicate linux/amd64 or linux/arm64.